tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 2.4
-envlist = {py36,py38}-{postgresql,mysql}{,-file,-swift,-ceph,-s3},pep8
+envlist = {py36,py38,py39}-{postgresql,mysql}{,-file,-swift,-ceph,-s3},pep8
 skipsdist = True
 
 [testenv]
```

Comment:
need to add py39 in some more places in this file below

Reply:
Good catch. Fixed in the latest version.